### ISSUE TITLE
Date Preprocessing

### ISSUE THREAD

**Issue by zhucq** [haoxiangsnr/FullSubNet#39]
Hi,

I use the pretrain model and its always very good effect except some special case, fine tuning need in my case.
Now I train the model like this:
1, download dns interspeech2020 branch data, split clean data to 6s length and 3s overlap, then every epoch model meet 500h noisy data;
2, I use 2080Ti * 8 and batch size set to 8. Other param just like train.toml;
3, the train loss look like this
![image](https://user-images.githubusercontent.com/39935745/147431964-46345821-4f37-4779-a57f-3ae9d6cbfc5e.png)
Is there any other preprocess on the orign train data?  And any advice for train in this little batch? 
Thank you

**Comment by hungcaovu** [comment]
@zhucq What was the issue ? I am facing the same problem. Could you give me some hints?
Thanks,
